packages/flutter/test/widgets/autocomplete_test.dart

```diff
@@ -3208,4 +3208,44 @@ void main() {
     expect(lastOptions, altEleOptions);
     expect(find.byKey(optionsKey), findsOneWidget);
   });
+
+  testWidgets('optionsLayerLink: open upward', (WidgetTester tester) async {
```

Comment:
Can you add an additional test that tests changing the optionsLayerLink passed to RawAutocomplete? I think you can probably do that using a StatefulBuilder and calling setState.